strings-csharp/HowToAddSeparatorsInThousandsPlaceForANumber/Tests/Tests.cs

```diff
@@ -0,0 +1,107 @@
+using HowToAddSeparatorsInThousandsPlaceForANumber;
+using System.Globalization;
+
+namespace Tests
+{
+    public class Tests
+    {
+        private readonly CultureInfo _englishCultureInfo = new("en-US");
+        private readonly CultureInfo _spanishCultureInfo = new("es-ES");
+
+        [Fact]
+        public void GivenNumericValue_WhenStringFormatIsUsed_ShouldReturnExpectedStringValue()
+        {
+            var message = NumbersFormatting.GetBalanceUsingTheStringFormatMethod();
+
+            Assert.Equal("Your account balance is: 2154002.535", message);
+        }
+
+        [Fact]
+        public void GivenNumericValue_WhenStringFormatWithCultureInfoIsUsed_ShouldReturnExpectedStringValue()
+        {
+            var message = NumbersFormatting.GetBalanceUsingTheStringFormatMethodAndACultureInfo(_englishCultureInfo);
+
+            Assert.Equal("Your account balance is: 2154002.535", message);
+        }
+
+        [Fact]
+        public void GivenNumericValue_WhenStringFormatWithCultureInfoAndNumericSpecifierIsUsed_ShouldReturnExpectedStringValue()
+        {
+            var message = NumbersFormatting.GetBalanceUsingTheStringFormatMethodACultureInfoAndSpecifiers(null, _englishCultureInfo);
+
+            Assert.Equal("Your account balance is: 2,154,002.54", message);
+        }
+
+        [Fact]
+        public void GivenNumericValue_WhenStringFormatWithCultureInfoNumericSpecifierAndDecimalCountIsUsed_ShouldReturnExpectedStringValue()
+        {
+            var message = NumbersFormatting.GetBalanceUsingTheStringFormatMethodACultureInfoAndSpecifiers(4, _englishCultureInfo);
+
+            Assert.Equal("Your account balance is: 2,154,002.5350", message);
+        }
+
+        [Fact]
+        public void GivenNumericValue_WhenStringFormatWithCultureInfoNumericSpecifierAndDecimalCount0IsUsed_ShouldReturnExpectedStringValue()
```

Comment:
@biagianina 
Some of your test names have `Count0`. Can this be corrected?

Reply:
@elfico Hi, I've changed the namings in several project parts. Now they are readable and specific, I think.